### PR TITLE
Tickets/sitcom 459

### PR DESCRIPTION
**Pull request by patrickingraham**

### CODE REVIEW

File: index.rst [hunk 1]
```diff
@@ -4,22 +4,215 @@
...
+It is anticipated that very few SIT-Com generated artifacts will fall into this category.
+
+SIT-Com Level Change Control
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In the case of documentation that is applicable to only SIT-Com personnel and is internal to the SIT-Com subsystem(s), the implementation of a SIT-Com change board has been discussed but is not currently in place.
+Examples of documents that are SIT-Com specific but should be under change control include the documentation of workflows (like this one), management plans, and/or SIT-Com specific procedures that require sign-off by specific domain experts.
+Until such a system is put in place, these types of documentation exist as `Technotes`_.
+
+When required to be under stricter change controls, branch protection should be put in place.
```
Comment (tribeiro) [inline index.rst:43]: Wouldn't it be best to apply this to all tech notes? and require PRs reviews etc? it might be confusing to mix things together..

File: index.rst [hunk 2]
```diff
@@ -4,22 +4,215 @@
...
+The designation of specific reviewers can also be manually specified.
```
Comment (tribeiro) [inline index.rst]: Not sure I understand this. Reviewers are selected manually in any case. Are you trying to differentiate cases where it is clear who should be the reviewer and cases where it is not so clear?

File: index.rst [hunk 3]
```diff
@@ -4,22 +4,215 @@
...
+In the event that this requires higher-level SIT-Com repo permissions, please consult the ``#rubinobs-sitcom-docs`` Slack channel for support.
```
Comment (tribeiro) [inline index.rst]: repo -> repository (?) or maybe rephrase it slightly:

>In the event that contributions requires additional permission in the SIT-Com  GitHub repository, please consult the ``#rubinobs-sitcom-docs`` Slack channel for support.

Maybe add a link to the org?

File: index.rst [hunk 4]
```diff
@@ -4,22 +4,215 @@
...
+
+This section outlines the different types of documentation used by SIT-Com personel.
+
+Jira Tickets
+^^^^^^^^^^^^
+
+SIT-Com related work should follow the `SIT-Com specific Jira workflow <https://sitcomtn-023.lsst.io/>`_.
+Documentation in Jira tickets should be sufficiently detailed such that it addresses the description in the task.
+It is not expected to be consulted in the future, especially on a regular occurrence. 
+In many cases, specifically when characterization activities have taken place over a series of tickets, external documentation should be developed and linked to the ticket(s).
```
Comment (tribeiro) [inline index.rst]: By "linked to the ticket(s)" do you mean the documentation should point to the ticket or the ticket(s) should point to the documentation?

Maybe rephrase so it is more clear what you mean?

File: index.rst [hunk 5]
```diff
@@ -4,22 +4,215 @@
...
+The type of document to be created is case dependent, but will most likely be covered by `Technotes`_ and/or inclusion of a section in the `Obs-Ops Webspace`_.
```
Comment (tribeiro) [inline index.rst]: "case dependent" -> "case-dependent" (?)

File: index.rst [hunk 6]
```diff
@@ -4,22 +4,215 @@
...
+The creation of the documentation can be performed under one of the already opened tickets, or, a new ticket can be created that explicitly deals with the generation of documentation.
```
Comment (tribeiro) [inline index.rst]: "tickets, or, a new" -> "tickets or, a new" (?)

File: index.rst [hunk 7]
```diff
@@ -4,22 +4,215 @@
...
+
+
+Verification Documentation and Artifacts
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The Systems Engineering team uses a specific Jira project for its verification performance and tracking.
+Personnel looking for verification documentation should consult `LSE-160 <ls.st/LSE-160>`_.
+
+The verification and validation process also utilize a specific repository for the storing of their notebooks that act as official artifacts.
```
Comment (tribeiro) [inline index.rst]: "notebooks" -> "Jupyter notebooks"?

File: index.rst [hunk 8]
```diff
@@ -4,22 +4,215 @@
...
+This is further described in the `Verification And Validation Notebooks`_ subsection below.
+
+
+Technotes
+^^^^^^^^^
+
+Technotes are useful for documenting outcomes of specific tests and/or findings, proposing new functionality or architectures, and/or documenting specific usages that augment user documentation.
+They can also be used as logs and/or status of longer term analyses.
+An example of this is the technote on Building and and fitting the pointing model for the AuxTel (`TSTN-014 <https://tstn-014.lsst.io/>`_.)
```
Comment (tribeiro) [inline index.rst]: "Building and and fitting" -> "Building and fitting"

I think there's a nicer way to reference tech notes. Try using

```
:tstn:`014`
```

File: index.rst [hunk 9]
```diff
@@ -4,22 +4,215 @@
...
+In this case, there are chapters (seen on the left) showing the analysis from different runs and the results that were obtained.
+This is an appropriate use of a technote.
+
+Technotes should not be used in place of user-facing how-to documentation and/or procedures. 
+This is better addressed as part of the `Obs-Ops Webspace`_ or even an addition to the collection of `Collaborative Notebooks`_.
+
+Creation and Review of Technotes
+--------------------------------
+
+When creating a technote, a Jira ticket should be created following the `SIT-Com specific Jira workflow <https://sitcomtn-023.lsst.io/>`_, this is primarily to facilitate an easier review process and to track progress.
```
Comment (tribeiro) [inline index.rst]: This phrase reads a bit weird. It has "creating" ... "created" in the first part and in the second part the "facilitate an easier"  seems a bit redundant. Maybe something like:

> When working on a new technote, or editing an existing one, start by creating a Jira ticket in the `SIT-Com specific Jira workflow <https://sitcomtn-023.lsst.io/>`_, this allows us to track the work and facilitates the review process.

File: index.rst [hunk 10]
```diff
@@ -4,22 +4,215 @@
...
+Ideally, reviews of technotes should be performed by the intended customer of the content.
```
Comment (tribeiro) [inline index.rst]: How about?

>In general, technote reviews should be performed by the intended customer of its content.

File: index.rst [hunk 11]
```diff
@@ -4,22 +4,215 @@
...
+Depending on how the document will be used and/or interacted with, there may be a preference for a specific format.
 
 .. note::
 
-   **This technote is a work-in-progress.**
+   Adding a ``\v`` to the url will display the rendered versions of all branches in the repo (e.g. ``sitcomtn-###.lsst.io/v``).
 
-Abstract
-========
+When applicable, technotes should also link and/or include all applicable supporting documentation.
```
Comment (tribeiro) [inline index.rst]: applicable used twice.. hehe... Maybe "When relevant"?

File: index.rst [hunk 12]
```diff
@@ -4,22 +4,215 @@
...
+This includes any notebooks and/or the tickets where work was performed, especially if there exists content in the ticket which may be applicable to the content.
```
Comment (tribeiro) [inline index.rst]: applicable again.. hehe

File: index.rst [hunk 13]
```diff
@@ -4,22 +4,215 @@
...
+If applicable, the replacement document(s) should be linked.
+This way the content is never entirely lost as it will remain in the git history of the repository.
+
+The Data Management team uses technotes extensively, more information on their using of the tool(s) are found at `their dedicated webpage <https://developer.lsst.io/project-docs/technotes.html>`_.
+
+
+Jupyter Notebooks
+^^^^^^^^^^^^^^^^^
+
+Jupyter notebooks (henceforth referred to as notebooks) are used routinely during commissioning exercises for both analysis and even for certain early observatory control sequences.
```
Comment (tribeiro) [inline index.rst:121]: Maybe play the "(henceforth referred to as notebooks)" card earlier?

File: index.rst [hunk 14]
```diff
@@ -4,22 +4,215 @@
...
+The use of them is not strictly required, however, the environment permits the simultaneous control of observatory functionality, data reduction/analysis tasks and documentation and is supported by the project.
```
Comment (tribeiro) [inline index.rst]: This phrase is a bit weird. Consider something like:

> Although their use is not strictly required, they allow simultaneously controlling of observatory functionality, data reduction/analysis tasks and documentation, and is supported by the project at large.

File: index.rst [hunk 15]
```diff
@@ -4,22 +4,215 @@
...
+
+The Data Management team uses technotes extensively, more information on their using of the tool(s) are found at `their dedicated webpage <https://developer.lsst.io/project-docs/technotes.html>`_.
+
+
+Jupyter Notebooks
+^^^^^^^^^^^^^^^^^
+
+Jupyter notebooks (henceforth referred to as notebooks) are used routinely during commissioning exercises for both analysis and even for certain early observatory control sequences.
+The use of them is not strictly required, however, the environment permits the simultaneous control of observatory functionality, data reduction/analysis tasks and documentation and is supported by the project. 
+They are a natural starting point for development of ideas and demonstrating proof of concept(s).
```
Comment (tribeiro) [inline index.rst]: Furthermore, they are...

**Review by tribeiro — APPROVED** [review]
Looks good! I added some inline comments, feel free to adopt what you feel is "applicable" (hehe).. 


Also, maybe we should consider mentioning the tssw develop guide? Maybe after the Obs-Control Webscape? This page has the information to contribute to the control software. I am working with Wouter to revamp it.

File: index.rst (hunk 1 above)
Comment (patrickingraham) [reply index.rst:43]: My hope is that since it's a jira ticket that the we can make the jira review process essentially force the review of the document in order to close out the task. 
I did add a note saying that the use of branches, PRs, and proper reviews is encouraged.

File: index.rst (hunk 2 above)
Comment (patrickingraham) [reply index.rst]: i meant it is possible to setup branch protection in a way that specific named individuals *have* to be the reviewer, and it can't just be anyone in the GH org.
I added "names" which hopefully clarifies this.

File: index.rst (hunk 3 above)
Comment (patrickingraham) [reply index.rst]: it's not the contributions that require support but the addition of specific branch protections... I think part of the mis-understanding here is related to the previous comment.
I tried to clarify a bit.

File: index.rst (hunk 5 above)
Comment (patrickingraham) [reply index.rst]: oh boy... google just confused me so i ended up just rephrasing to avoid this...

File: index.rst (hunk 6 above)
Comment (patrickingraham) [reply index.rst]: apparently we were both wrong...
"tickets, or a new..."

File: index.rst (hunk 9 above)
Comment (patrickingraham) [reply index.rst]: Good point. I didn't use your example but I did modify it.

File: index.rst (hunk 13 above)
Comment (patrickingraham) [reply index.rst:121]: Doesn't quite work... but I did add a "Jupyter" further up to relieve the possibility of confusion.

**Comment by patrickingraham** [comment]
I added a plug for the developer guides.
Thanks for the review!
